src/test/java/io/supertokens/test/totp/TOTPRecipeTest.java

```
@@ -167,7 +167,7 @@ public void createDeviceAndVerifyCodeTest() throws Exception {
         TOTPDevice device = Totp.registerDevice(main, "user", "device1", 1, 1);
 
         // Try login with non-existent user:
-        assertThrows(TotpNotEnabledException.class,
+        assertThrows(InvalidTotpException.class, // TODO: Can we throw a better error?
                 () -> Totp.verifyCode(main, "non-existent-user", "any-code", true));
```

Can we throw a better error? Maybe `UnknowDeviceError`?

---

We are doing `UnknownUserIdError`. Done.